packages/appstore/src/components/app.tsx

```diff
@@ -2,6 +2,7 @@ import classNames from 'classnames';
 import * as React from 'react';
 import { observer } from 'mobx-react-lite';
 import { setWebsocket, routes } from '@deriv/shared';
+import { StoreProvider } from '@deriv/stores';
 import Routes from 'Components/routes/routes';
 import { useStores, initContext } from 'Stores';
```

Comment:
don't we need to change this local useStores() into useStore() which is imported from '@deriv/stores' instead?

Reply:
@mahdiyeh-fs Yes as I mentioned [here](https://github.com/binary-com/deriv-app/pull/7108#discussion_r1041789413), We need to replace the existing `useStores` solution in `@deriv/appstore` with the one from `@deriv/stores`.

It is out of this PR scope to make these changes as here I'm just trying to provide a solution for the raised issue, Each team will start to replace `connect` with `useStore` (`useStores` with `useStore` in `@deriv/appstore` case) in separate cards as we are already doing it in `@deriv/cashier`.

---

I will mark it as `deprecated` for now 🙇🏻